lib/registry.sh

```diff
@@ -13,8 +13,8 @@ function create-dev-registry {
   )
 
   local host_ip="${HOST_IPADDR:-$(get-machine-ip)}"
-  export DEV_REGISTRY="${host_ip}:5000"
-  save-vars DEV_REGISTRY
+  export SUGGEST_DEV_REGISTRY="${host_ip}:5000"
+  save-vars SUGGEST_DEV_REGISTRY
```

Comment:
There were some times when you wanted to pull prebuilt Deis components from places other than the local registry when playing around on Vagrant. This change supports that workflow.